pysite/views/main/redirects/gitlab.py

```diff
@@ -4,5 +4,5 @@
 class GitLabView(RedirectView):
     path = "/gitlab"
     name = "gitlab"
-    page = "https://gitlab.com/python-discord/"
+    page = "https://github.com/python-discord/"
```

Comment:
This view intentionally points to GitLab, we already have another view which redirects to GitHub.

Reply:
@jchristgit why? we already have the GitHubView

Reply:
I meant, change the path as in remove it completely